README.rst

```diff
@@ -64,14 +64,20 @@ Launch an interpreter with ``requests``, ``flask`` and ``psutil`` in the environ
 
     $ pex requests flask 'psutil>2,<3'
 
+Save Dependencies From Pip
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
 Or instead freeze your current virtualenv via requirements.txt and execute it anywhere:
 
 .. code-block:: bash
 
-    $ pex $(pip freeze) -o my_virtualenv.pex
+    $ pex -r requirements.txt -o my_virtualenv.pex
```

Comment:
Here you're sensibilites are trampling others. The `$(pip freeze)` was both contributed by another non-Pex developer trying to clarify docs _and_ -r requirements.txt may not exist in your virtualenv making the lead-in sentence in combination with this example not make sense.